dist/index.html

```diff
@@ -19,12 +19,12 @@ <h2 class='goals-header'>Goals</h2>
        
         <section class='widget-category'>
 
-          <section class='widget'>
```

Comment:
looks awesome, maybe we need try to avoid using section element , and maybe instead use fomwhere article, nav, form